src/Kunstmaan/AdminBundle/Helper/Security/Acl/AclHelper.php

```diff
@@ -55,6 +60,7 @@ class AclHelper
     public function __construct(EntityManager $em, TokenStorageInterface $tokenStorage, RoleHierarchyInterface $rh, $permissionsEnabled = true)
     {
         $this->em = $em;
+        $this->databasePlatform = $this->em->getConnection()->getDatabasePlatform()->getName();
```

Comment:
When the query is not duplicated any more I would inline this check in the query method, there is no need to have this as a class property

Reply:
Good point ! done

Reply:
Agree, done